db/fixtures/miq_policy_sets.yml

```diff
@@ -105,7 +105,24 @@
           description: Initiate SmartState Analysis for Container Image
           action_type: default
           options: {}
-      Condition: []
+      Condition:
+        - name: do not scan image-inspectors image
+          description: Dont scan image-inspectors image
+          modifier: deny
+          expression: !ruby/object:MiqExpression
+            exp:
+              ENDS WITH:
```

Comment:
IIUC the name has values like `openshift/image-inspector`.
So yes, you want ENDS WITH but can probably tighten it to `value: "/image-inspector"`